packages/smooth_app/ios/Podfile.lock

```diff
@@ -5,6 +5,8 @@ PODS:
     - Flutter
   - camera_avfoundation (0.0.1):
     - Flutter
+  - camerawesome (0.0.1):
```

Comment:
`camerawesome` is not supposed to be there anymore: please get the latest version of the code first.

Reply:
That was weird, because I was on `develop`.
But I've relaunch the command and pushed a new commit